controllers/carsController.js

```diff
@@ -5,6 +5,7 @@ module.exports = (connection) => {
   const controllerMethods = {};
 
   controllerMethods.addCarModification = async (req, res) => {
+
```

Comment:
why empty line? I see in file below in the same situation you have removed empty line. Be consistent